readthedocs/api/v2/serializers.py

```diff
@@ -133,11 +136,63 @@ class Meta:
         exclude = []
 
 
+class BuildCommandReadOnlySerializer(BuildCommandSerializer):
+
+    """
+    Serializer used on GETs to trimm the commands' path.
```

Comment:
```suggestion
    Serializer used on GETs to trim the commands' path.
```

I trimmed trimm to trim :)